python/tritonserver/_c/tritonserver_pybind.cc

```diff
@@ -1434,6 +1434,11 @@ class PyServer : public PyWrapper<struct TRITONSERVER_Server> {
     owned_ = true;
   }
 
+  uintptr_t get_c_ptr()
```

Comment:
```suggestion
  uintptr_t _ptr()
```
Would like to avoid ```get``` to keep it parallel with other APIs. Do we need this API or could the caller do the cast?